api/quotes.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Quote = require('../models/quote');
+const router = require('express').Router();
+const HTTPStatus = require('http-status');
+
+router.get('/quotes', listQuotes);
+router.get('/quotes/:id', getQuote);
+router.post('/quotes', createQuote);
+router.delete('/quotes/:id', deleteQuote);
+router.put('/quotes/:id', updateQuote);
```

Comment:
In previous routers you ordered routes per endopoints then methods so do it here as well.